cli/js/compiler.ts

```diff
@@ -1327,7 +1184,6 @@ async function compile(
       rootNames,
       options,
       host,
-      oldProgram: TS_SNAPSHOT_PROGRAM,
```

Comment:
@kitsonk I removed `oldProgram` after your comments - it has marginal impact on performance (0.04s)

Reply:
👍 as I said above, we should document why we have an unused const above.